code/modules/vehicles/mecha/mech_melee_attack.dm

```diff
@@ -70,9 +70,9 @@
 		switch(mecha_attacker.damtype)
 			if(BRUTE)
 				if(mecha_attacker.force > 35) // durand and other heavy mechas
-					Unconscious(20)
+					Unconscious(0)
```

Comment:
You need to at least mark that this is an edit, control f the codebase with "LETHAL EDIT" or it might be "LETHALSTATION EDIT" to see how I've been doing it

Reply:
Should be fixed!